src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -17,20 +17,30 @@ const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ fagsak }) =
     const [errorMessage, setErrorMessage] = React.useState<string | undefined>(undefined);
 
     React.useEffect(() => {
-        hentAktivVedtaksbrev(fagsak)
-            .then((response: Ressurs<string>) => {
-                if (response.status === RessursStatus.SUKSESS) {
-                    setBrev(response.data);
-                    setErrorMessage(undefined);
-                } else if (response.status === RessursStatus.FEILET) {
-                    setErrorMessage(response.melding);
-                } else {
-                    setErrorMessage('Ukjent feil, kunne ikke generere forhåndsvisning.');
-                }
-            })
-            .catch(error => {
-                setErrorMessage('Ukjent feil, Kunne ikke generere forhåndsvisning.');
-            });
+        if (
+            fagsak?.behandlinger
```

Comment:
I dont think we need the questionmarks. I have not seen this in typescript before, what is it called?

Reply:
It is called "optional chain", used to simplify validation of expressions.
See 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
for details📖

Reply:
Riight, my vscode used an early ts version and gave me some nasty errors 👎  Looks good!